models/interval.go

```diff
@@ -14,8 +14,8 @@ var (
 	IntervalPast7DaysYesterday = &IntervalKey{"Last 7 Days from Yesterday"}
 	IntervalPast14Days         = &IntervalKey{"Last 14 Days"}
 	IntervalPast30Days         = &IntervalKey{"30_days", "last_30_days", "Last 30 Days"}
-	IntervalPast12Months       = &IntervalKey{"12_months", "last_12_months"}
-	IntervalAny                = &IntervalKey{"any"}
+	IntervalPast12Months       = &IntervalKey{"12_months", "last_12_months", "last_year"}
```

Comment:
Thanks for the contribution. 
Do you know if `last_year` refers to the last 12 months from today, or is it actually the past year (from Jan to Dec)?

Reply:
The `last_year`refer to the last 12 months from today.
<img width="229" alt="Screenshot 2022-08-17 at 10 13 11" src="https://user-images.githubusercontent.com/68403576/185069020-25a9da88-8163-4cc6-8c42-3b7ebf03e709.png">